### PR TITLE
Add Deque opportunity (#542)

### PR DESCRIPTION
**Pull request by stabee**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -125,6 +125,15 @@
     "dateAdded": "2019-10-23T00:39:55.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Deque",
+    "difficulty": "easy",
+    "description": "Fill out the sign up form with your name, email, job title, and company name. Then, respond to Deque's feedback survey",
```
Comment (aslafy-z) [inline data.json:131]: No need to mention all the fields to fill, and the description misses the outcome. Answer the survey, and then?
Also, I think it's not that fair to push user to sign up and answer directly. 

Answer Deque's feedback survey once you're logged in to get an awesome t-shirt.

File: data.json [hunk 2]
```diff
@@ -125,6 +125,15 @@
     "dateAdded": "2019-10-23T00:39:55.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Deque",
+    "difficulty": "easy",
+    "description": "Answer Deque's feedback survey once you're logged in to get an awesome t-shirt.",
+    "reference": "https://twitter.com/dequesystems/status/1182651282869362694",
+    "image": "https://i.imgur.com/t82DlcW.jpg",
+    "dateAdded": "2020-03-10T00:00:00.000Z",
```
Comment (aslafy-z) [inline data.json:134]:
```suggestion
    "dateAdded": "2020-02-25T14:33:22.000Z",
```

**Comment by aslafy-z** [comment]
Thank you @stabee! It's now live at https://devswag.io! :partying_face:

**Comment by stabee** [comment]
Nice, thank you! :)

**Pull request merged**